src/utils/filter-children/filter-children.util.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const renderChild = (child, callback) => {
+  if (callback) return callback(child);
+  return child;
+};
+
+const defaultFilter = (processedText, processedValue, filterType) => {
```

Comment:
It might be better to change the signature of this so the `defaultFilter` is just a plain function. You then don't need to pass in both `filterType` and `filter` into `filterChildren`.

Reply:
done 👍